app/components/SideBar/SideBar.module.css

```diff
@@ -29,7 +34,13 @@
   position: relative;
   overflow-y: auto;
   overflow-x: hidden;
-  flex: 1;
+  display: flex;
+  flex-direction: column;
+}
+
+.sidebar.sidebarReduced .sidebarMain,
+.sidebar.sidebarReduced .sidebarScroll {
+  overflow: visible;
```

Comment:
By adding this ``CSS`` property at these components, we guarantee that their contents (tooltips here) won't be clipped by them. Otherwise, ``overflow: hidden`` would not be overrided and the we'd have this result:

![image](https://user-images.githubusercontent.com/39631429/97127296-01b19000-1718-11eb-8803-153d5655c461.png)

Reply:
hmmmm nice, mate! Thanks for explaining :)